client/src/axiom/utils.ts

```diff
@@ -67,24 +80,32 @@ export function argsObjToArr(
   ]
 }
 
-export async function getMaxFeePerGas(axiom: AxiomSdkCore): Promise<string> {
+export async function getMaxFeePerGas(axiom: AxiomSdkCore, overrides?: AxiomV2ClientOverrides): Promise<string> {
+  const axiomQueryAddress = overrides?.queryAddress ?? getAxiomV2QueryAddress(axiom.config.chainId.toString());
```

Comment:
here and in many other places, the `mock?` is not being set so it's always false. 
this only works because https://github.com/axiom-crypto/axiom-sdk-client/blob/035473fae9811c0b226c6bea16f1137dc4431482/client/src/lib/address.ts#L24 is set.
We should unset this for accuracy, and then there should be some more systematic way to handle this.

Is mock only used to determine the default query address?

Reply:
@jonathanpwang i've pushed a change to not handle mock and just have an AxiomV2Query address per source-target chain pair. Any other addresses can be handled w/ override. this should simplify things a bunch